Add a coreml conversion script (#14)
TwentyBN/20bn-realtimenet: README.md

From our experiments, our estimates correlate well with the workout intensity (intense workouts burn more calories) so, regardless of the absolute accuracy, it should be fair to use this metric to compare one workout to another.

## Running on an iOS Device and CoreML Conversion

> If you're interested in mobile app development and want to run our models on iOS devices, please check out [20bn-realtimenet-ios](https://github.com/TwentyBN/20bn-realtimenet-iOS).
> This other repo provides step by step instructions on how to get our gesture demo to run on an iOS device.

Reviewer: ```suggestion
If you're interested in mobile app development and want to run our models on iOS devices, please check out [20bn-realtimenet-ios](https://github.com/TwentyBN/20bn-realtimenet-iOS) for step by step instructions on how to get our gesture demo to run on an iOS device.
```
